ophyd/pseudopos.py

```diff
@@ -675,14 +665,6 @@ def real_position(self):
         '''Real motor position namedtuple'''
         return self.RealPosition(*self._real_cur_pos.values())
 
-    @property
```

Comment:
I think this should be saved. Pseudopositioners are potentially multi-dimensional, with a dynamic number of dimensions, and this handles that specially.